react/__test__/index.test.tsx

```diff
@@ -225,5 +227,14 @@ teste|abc
     expect(component).toBeDefined()
     expect(component.asFragment()).toMatchSnapshot()
   })
+
+  it('should sanitize the font prop', () => {
+    const typography = 't-heading-1'
+    const { container } = render(<RichText {...defaultProps} text="foo" font={`${typography} foo`} />)
+
+    const element = container.querySelector(`.${typography}`)
+
+    expect(element).toBeDefined()
```

Comment:
Won't this always be true? `querySelector()` returns the matched element or `null` if there are no matches, which `toBeDefined()` considers a defined value, since it's not `undefined`.

Reply:
wow, I didn't know about that! Nice catch